### ISSUE TITLE
Status bar black-out on iOS 15.1

### ISSUE THREAD

**Issue by stenogriz** [holzschu/Carnets#205]
In notebook viewing mode iOS status bar is being blacked out (see attached screenshots), though in line edit mode or notebook running mode everything is fine.
Setup:
iOS 15.1 running on iPhone 6s.
Carnets plus version 1.6.1.
Info from Carnets 'about' section:
Version of notebook server 7.0.0dev0
Running on Python 3.9.2+
![2021-11-08 08-32-12](https://user-images.githubusercontent.com/3997288/140690937-54e930ba-9fc2-4a1c-8795-e7ea6b76b4a3.PNG)
![2021-11-08 08-31-59](https://user-images.githubusercontent.com/3997288/140690969-839e88cc-1d56-4b0a-939e-08624f566866.PNG)

**Comment by holzschu** [comment]
Indeed, it does black out. It seems to depend on how the notebook is scrolled, and it is one of the weird details happening: I've also noticed that scrolling up zooms on the notebook, and that predictive text is present, even though it shouldn't. I'm going to have a look, but these seem to be iOS-related issues, for which I have little control.
